### PR TITLE
More improvements

### PR DESCRIPTION
**Pull request by leomoon**
- fixes php 8.1 null value warning
- Enables table of content and typographer in ParsedownExtended
  These two are disabled by default.
  - Add table of content to a doc: [toc]
    https://benjaminhoegh.github.io/ParsedownExtended/docs/table%20of%20content.html
  - Typographer replaces some predefined characters:
    https://benjaminhoegh.github.io/ParsedownExtended/docs/typographer.html
- adds mermaid 9.4.3 to support different kind of charts:
  Mermaid supports:
  - sequence diagrams
  - flowcharts
  - Gantt charts
  - class and state diagrams
  - pie charts
    Example chart:
    ```mermaid
    graph TD;
        A-->B;
        A-->C;
        B-->D;
        C-->D;
    ```
- adds nginx configuration to README
- adds persian language

### CODE REVIEW

File: localizations/fa.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+	"Language": "فارسی",
```
Comment (Zavy86) [inline localizations/fa.json:2]: this is amazing ;)

**Comment by Zavy86** [comment]
a curiosity.
are you using wikidocs on some public site?

**Comment by Zavy86** [comment]
for mermaid what is the tag?

File: localizations/fa.json (hunk 1 above)
Comment (leomoon) [reply localizations/fa.json:2]: I'll ask my friends to do more translations in their native language. :D

**Comment by leomoon** [comment]
I have WikiDocs local only accessed via `wiki.local`.
so mermaid would be like a code tag with mermaid like:
\```mermaid
    graph TD;
        A-->B;
        A-->C;
        B-->D;
        C-->D;
\```

**Comment by leomoon** [comment]
You can also do mind maps like:
\```mermaid
mindmap
  root((mindmap))
    Origins
      Long history
      ::icon(fa fa-book)
      Popularisation
        British popular psychology author Tony Buzan
    Research
      On effectiveness<br/>and features
      On Automatic creation
        Uses
            Creative techniques
            Strategic planning
            Argument mapping
    Tools
      Pen and paper
      Mermaid
\```

**Comment by Zavy86** [comment]
very good https://demo.wikidocs.it/aaaa

**Comment by leomoon** [comment]
And supports both dark and light themes. :D

**Comment by Zavy86** [comment]
end of the month.. release 1.0.0! :)

**Pull request merged**